### DISCUSSION TITLE
A post about basic inverse kinematics in mujoco.

### DISCUSSION THREAD

**Discussion by alefram** [google-deepmind/mujoco#1419] [Show and tell]
I was learning about inverse kinematics and using Mujoco to practice it, so I created a write-up about my experience. Maybe some people can benefit from it. If there are any errors or areas for improvement, please let me know so I can update it. Thanks!

**Comment by alefram** [comment]
https://alefram.github.io/posts/Basic-inverse-kinematics-in-Mujoco

**Reply by yuvaltassa** [reply]
This is very nice. One thing I think this is missing is the interactive/smooth setting. It's actually kind or rare that you just want to jump to your desired pose. Usually you want a smooth trajectory that takes you from the current pose to the desired one.

@kevinzakka, you were working a nice interactive inverse kinematics thing, you probably have something to say?

**Reply by kevinzakka** [reply]
Here's some code: https://github.com/kevinzakka/mjctrl

**Reply by alefram** [reply]
Thanks @kevinzakka I will check it.

**Reply by alefram** [reply]
I have updated the post. Thank you for your help. @kevinzakka @yuvaltassa

**Comment by alefram** [comment]
@yuvaltassa Yeah, I was thinking about including it in this post or another. I am currently checking out the robotics manipulation techniques of [Russ Tedrake](https://manipulation.csail.mit.edu/pick.html#jacobian) course to implement it, maybe some control. Do you have any advice?

**Reply by yuvaltassa** [reply]
My advice is look at @kevinzakka's code 🙂

**Reply by songwookim** [reply]
> @yuvaltassa Yeah, I was thinking about including it in this post or another. I am currently checking out the robotics manipulation techniques of [Russ Tedrake](https://manipulation.csail.mit.edu/pick.html#jacobian) course to implement it, maybe some control. Do you have any advice?

Thanks for your post. it was really helpful for me.

**Reply by alefram** [reply]
@songwookim Thanks I like to share when I learning mujoco, if there is any advice would appreciated.
